.github/workflows/homebrew.yaml

```diff
@@ -1,13 +1,11 @@
 # Test installation of our homebrew tap for latest version
 name: Homebrew
 on:
-  schedule:
-    # run daily 20:00 on main branch
-    - cron: '0 20 * * *'
```

Comment:
since this is a package test whose result is not changed by commits to main this should not be changed